modules/RouteUtils.js

```diff
@@ -10,7 +10,14 @@ export function isReactChildren(object) {
 }
 
 function checkPropTypes(componentName, propTypes, props) {
-  componentName = componentName || 'UnknownComponent'
+  componentName = {} || ''
```

Comment:
What is this supposed to do?

This always assigns `{}` to componentName, and the function as a whole will always return `{}`, as @ryanflorence noted.